geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/SortedSetRangeOptions.java

```diff
@@ -20,10 +20,10 @@
 import java.util.Arrays;
 
 public class SortedSetRangeOptions {
-  private final double minDouble;
-  private final boolean minExclusive;
-  private final double maxDouble;
-  private final boolean maxExclusive;
+  private final double startDouble;
+  private final boolean startExclusive;
+  private final double endDouble;
+  private final boolean endExclusive;
```

Comment:
Rather than renaming all of these fields and methods, would it be simpler to just pass the appropriate arguments to the constructor in `ZRevRangeByScoreExecutor`? That is, if we know that `commandElements.get(2)` is the max, then use that as the second argument in the constructor rather than the first, using the `isRev()` method to switch the behaviour.